nucleus/autocurate.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+import requests
+from nucleus.constants import (
+    JOB_CREATION_TIME_KEY,
+    JOB_LAST_KNOWN_STATUS_KEY,
+    JOB_TYPE_KEY,
+)
+from nucleus.job import AsyncJob
+
+
+def entropy(name, model_runs, client):
+    model_run_ids = [model_run.model_run_id for model_run in model_runs]
+    dataset_id = model_runs[0].dataset_id
```

Comment:
assert len({_.dataset_id for _ in model_runs}) == 0, f"Model runs have conflicting dataset ids: {model_runs}"